hw3/Task_12.ts

```diff
@@ -0,0 +1,25 @@
+//Task 12
+// Написать программу, которая выполняет следующие операции:
+// - Получает возраст пользователя, если он больше 18, то выводит сообщение - `Попей пивка`.
+// - Если меньше, то выводит сообщение - `Пей колу`.
+// - Добавить условие, что если возраст от 16-18, выводим сообщение -
+// `Можешь выкурить сигаретку, только маме не говори`.
+
+let myAge: number = 17;
+if (myAge >= 18) {
+    console.log('Попей пивка')
+} else if (myAge >= 16 && myAge < 18) {
+    console.log('Можешь выкурить сигаретку, только маме не говори');
+    console.log('Пей колу')
+} else {
+    console.log('Пей колу')
+}
+
+// Почему не срабатывает в др порядке?
+// if (myAge >= 18) {
+//     console.log ('Попей пивка')
+// } else if (myAge < 18) {
+//     console.log ('Пей колу')
+// } else if (myAge >= 16 && myAge < 18) {
+//     console.log('Можешь выкурить сигаретку, только маме не говори') // не доходит до этого цикла
```

Comment:
this happens because in case you enter somewhere in it-else block execution won't check another blocks

Reply:
Got it, thanks.